File: food-ordering-system/restaurant-service/src/main/java/mk/finki/restaurantservice/infrastructure/RestaurantJpaRepository.java
package mk.finki.restaurantservice.infrastructure;

import mk.finki.restaurantservice.domain.Restaurant;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import java.util.UUID;

@Repository
public interface RestaurantJpaRepository extends JpaRepository<Restaurant, UUID> {
}

File: food-ordering-system/restaurant-service/src/main/kotlin/mk/finki/restaurantservice/infrastructure/persistence/RestaurantJpaRepository.kt
package mk.finki.restaurantservice.infrastructure.persistence

import mk.finki.restaurantservice.domain.restaurant.Restaurant
import mk.finki.restaurantservice.domain.restaurant.valueobjects.RestaurantId
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RestaurantJpaRepository : JpaRepository<Restaurant, RestaurantId>
